shiny/driver/androiddriver/screen.go

```diff
@@ -0,0 +1,77 @@
+//go:build android
+// +build android
+
+package androiddriver
+
+import (
+	"image"
+	"image/color"
+
+	"github.com/oakmound/oak/v3/shiny/driver/internal/event"
+	"github.com/oakmound/oak/v3/shiny/screen"
+	"golang.org/x/image/draw"
+	"golang.org/x/image/math/f64"
+	"golang.org/x/mobile/app"
+	"golang.org/x/mobile/event/size"
+	"golang.org/x/mobile/exp/gl/glutil"
+	"golang.org/x/mobile/geom"
+	"golang.org/x/mobile/gl"
+)
+
+var _ screen.Screen = &screenImpl{}
+
+type screenImpl struct {
+	event.Deque
+
+	app   app.App
+	glctx gl.Context
+
+	images       *glutil.Images
+	activeImages []*imageImpl
+
+	lastSz size.Event
+}
+
+func (s *screenImpl) NewImage(size image.Point) (screen.Image, error) {
+	img := &imageImpl{
+		screen: s,
+		size:   size,
+		img:    s.images.NewImage(size.X, size.Y),
+	}
+	s.activeImages = append(s.activeImages, img)
```

Comment:
Question we can have simultaneous access to this do  we care enough to put a lock around it? Or given that the only other accessor is at lifecycle cross event (cleanup) we just dont care.